.github/workflows/mosaiq-db-tests.yml

```diff
@@ -5,17 +5,12 @@ on:
     branches:
       - main
   pull_request:
```

Comment:
this seems weird (to have the pull_request with a colon, but nothing below) -- is this the right syntax?

Reply:
Yeah, it is a bit weird. But yup, that's the syntax:

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration